scripts/benchmark-requirements.txt

```diff
@@ -12,6 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-benchadapt@git+https://github.com/conbench/conbench.git@44e81d1#subdirectory=benchadapt/python
-benchalerts@git+https://github.com/conbench/conbench.git@44e81d1#subdirectory=benchalerts
-benchclients@git+https://github.com/conbench/conbench.git@44e81d1#subdirectory=benchclients/python
+benchadapt
+benchalerts
+benchclients
```

Comment:
cool, mental note to check on these builds after our releases to PyPI!

Reply:
We can also pin to the current version to be safe but the previously pinned version was like a year old